src/teamcity.py

```diff
@@ -1,106 +0,0 @@
-from getpass import getpass
```

Comment:
what on earth was all this?

Reply:
not entirely sure, something to do with fetching artifacts that was not being currently used.